xep-0405.xml

```diff
@@ -199,7 +208,8 @@ This approach  enables flexible support of multiple clients for a MIX channel pa
     to='hag66@shakespeare.example/UUID-c8y/1573'
     type='result'>
   <query xmlns='http://jabber.org/protocol/disco#info'>
-    <feature var='urn:xmpp:mix:pam:0'/>
+    <feature var='urn:xmpp:mix:pam:1'/>
+     <feature var='urn:xmpp:mix:pam:1:archive'/>
```

Comment:
[trivial] Here's one space too much

Reply:
pedant!

Reply:
Sorry, but it really doesn't look too good when the indentation is wrong.

Reply:
I know!  I fixed it.  Thanks for pointing it out